examples/iotivity_examples/client_switch/client_switch.c

```diff
@@ -76,6 +76,7 @@ stop_observe(void *data)
     return DONE;
 }
 
+#ifdef BUTTON_GPIO
```

Comment:
I may have missed something but it's totally unclear to me why this is required : the code of this function has nothing to do with the board definitions.